tmt/utils.py

```diff
@@ -200,8 +237,14 @@ def is_pidfile(cls, exit_code: Optional[int]) -> bool:
 DEFAULT_WAIT_TICK_INCREASE: float = 1.0
 
 # Defaults for GIT attempts and interval
+DEFAULT_GIT_CLONE_TIMEOUT: Optional[int] = None
+GIT_CLONE_TIMEOUT: Optional[int] = configure_optional_constant(None, 'TMT_GIT_CLONE_TIMEOUT')
```

Comment:
Did you want to use `DEFAULT_GIT_CLONE_TIMEOUT` defined above here instead of `None`?

Reply:
Correct, addressed in 7a245bef